orbit/pkg/packaging/windows.go

```diff
@@ -403,7 +403,7 @@ func SanitizeVersion(version string) ([]string, error) {
 	}
 	if len(vParts) == 3 && strings.Contains(vParts[2], "-") {
 		parts := strings.SplitN(vParts[2], "-", 2)
-		if len(parts) != 2 || parts[0] == "" || parts[1] == "" {
+		if len(parts) < 2 || parts[0] == "" || parts[1] == "" {
```

Comment:
Intended to allow versions like `orbit-v.1.2.3-1-some-notes-about-the-prerelease`, where previously the notes part would have caused an error.